packages/babel-generator/src/generators/types.js

```diff
@@ -4,16 +4,6 @@
 import * as t from "babel-types";
 
 export function Identifier(node: Object) {
-  // FIXME: We hang variance off Identifer to support Flow's def-site variance.
-  // This is a terrible hack, but changing type annotations to use a new,
-  // dedicated node would be a breaking change. This should be cleaned up in
-  // the next major.
-  if (node.variance === "plus") {
-    this.push("+");
-  } else if (node.variance === "minus") {
-    this.push("-");
-  }
```

Comment:
Feels like we need to have this for backward-compatibility. Should we do

    if (node.variance) return TypeParameter(node);